src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -188,7 +191,7 @@
 
 
 def get_privacy_request_or_error(
-    db: Session, privacy_request_id: str
+    db: Session, privacy_request_id: str, error_if_deleted: Optional[bool] = True
```

Comment:
If the privacy request is deleted, we error by default, unless the `error_if_deleted` argument is set to False